conda/common/url.py

```diff
@@ -7,11 +7,11 @@
 import re
 import socket
 
+from .compat import input, on_win
```

Comment:
This is overloading Python's `input` right?

Reply:
Yes. And that's exactly what ltgm was suggesting we to here.
https://lgtm.com/rules/3980089/